.github/workflows/continuous-integration-workflow.yml

```diff
@@ -53,8 +53,9 @@ jobs:
       - name: Set up JDK 11
         uses: actions/setup-java@v3
         with:
-          java-version: '11'
-          distribution: 'adopt'
+          java-version: '17'
```

Comment:
I think you are only wanting to add the `cache` line, correct? Please remove these other two changes.

Reply:
@jzheaux Thank you for your review! I pushed a commit.